test/solve-overview.spec.js

```diff
@@ -63,33 +64,6 @@ describe('Solve', () => {
         expect(this.wrapper.text()).to.contain('read');
       });
 
-      it('should display author when syndicated content is unchecked', async () => {
-        await this.wrapper.vm.getData();
-        const first = this.wrapper.vm.content[0];
-        first.field_tl_ = first.field_tl_.replace(this.wrapper.vm.tokens.syndicated, '');
-        // Check featured post.
-        expect(this.wrapper.text()).to.not.contain('Roger Avalos & HEB');
-        expect(this.wrapper.text()).to.contain('Mike Rustyellow');
-      });
-
-      it('should display author when syndicated content is null', async () => {
-        await this.wrapper.vm.getData();
-        const first = this.wrapper.vm.content[0];
-        first.field_tl_ = first.field_tl_.replace(this.wrapper.vm.tokens.syndicated, '');
-        // Check featured post.
-        expect(this.wrapper.text()).to.not.contain('Roger Avalos & HEB');
-        expect(this.wrapper.text()).to.contain('Mike Rustyellow');
-      });
-
-      it('should display publication attribution when syndicated content is checked', async () => {
-        await this.wrapper.vm.getData();
-        const first = this.wrapper.vm.content[0];
-        first.field_tl_ = `${first.field_tl_}:::${this.wrapper.vm.tokens.syndicated}`;
```

Comment:
So this line is actually manipulating the DOM at little bit, which was having some unintended affects on subsequent tests. So if you notice the counts are off by one following this test, it's because of this. We can safely adjust those counts by one.